src/main/scala/sigmastate/eval/RuntimeCosting.scala

```diff
@@ -1520,6 +1554,13 @@ trait RuntimeCosting extends SigmaLibrary with DataCosting with Slicing { IR: Ev
         val cost = inputC.cost + costOf(node)
         withDefaultSize(res, cost)
 
+      case ByteArrayToLong(In(arr)) =>
+        val arrC = asRep[Costed[Coll[Byte]]](arr)
+        val value = sigmaDslBuilder.byteArrayToLong(arrC.value)
+        val size = arrC.dataSize
```

Comment:
Since we know the size of resulting Long is constant, it is better to use withDefaultSize method here, this makes the costing graph simpler and the whole process faster.

Reply:
Thanks. Done.